source/minimessage.rst

```diff
@@ -228,9 +215,9 @@ Keybind
 Allows displaying the configured key for actions
 
 Tag
-   ``<key:_key_>``  
+   ``<key:_key_>``
 Arguments
-   * ``_key_``, the minecraft key of the action  
+   * ``_key_``, the minecraft key of the action
```

Comment:
```suggestion
   * ``_key_``, the keybind identifier of the action
```

or something similar using `keybind identifier` instead of `minecraft key`